site/src/components/BorderedMenu/BorderedMenu.tsx

```diff
@@ -2,7 +2,7 @@ import Popover, { PopoverProps } from "@material-ui/core/Popover"
 import { makeStyles } from "@material-ui/core/styles"
 import { FC, PropsWithChildren } from "react"
 
-type BorderedMenuVariant = "admin-dropdown" | "user-dropdown"
+type BorderedMenuVariant = "user-dropdown"
```

Comment:
do we still need this type?

Reply:
Good catch! I will clean up this. I'm planning to run clean up to find unused code and delete all of them. I'm going to do that this week.